### PR TITLE
推荐java编程思想

### PR DESCRIPTION
**Pull request by Balticbb**
希望作者采纳
reviewer @xuxiaodong @dofy @justjavac @jsann

### CODE REVIEW

**Comment by Balticbb** [comment]
java

**Comment by Bethannfrum** [comment]
Java

File: README.md [hunk 1]
```diff
@@ -13,6 +13,7 @@
 
 ![justjavac微信公众帐号](http://justjavac.com/assets/images/weixin-justjavac.jpg)
 
+## java编程思想
```
Comment (Bethannfrum) [inline README.md]: Java with images

Comment (Bethannfrum) [reply README.md]: (Http://justjavac.com/assets/imagines/jpg)

**Pull request closed without merging**